packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -170,6 +170,37 @@ class HttpRequestView extends StatelessWidget {
   }
 }
 
+/// A button for copying [DartIOHttpRequestData] contents.
+///
+/// If there is no content to copy, the button will not show. The copy contents
```

Comment:
instead of hiding the button, we should just disable the onPressed handler. This will keep the UI consistent and not confuse users why sometimes the button is there and sometimes it is not. This is pretty standard across the rest of DevTools. For example, we don't remove a pause button when we are already paused, we just disable the onPressed handler so it shows up as disabled.

Reply:
SGTM
Issue created: https://github.com/flutter/devtools/issues/5519